packages/strapi-parts/src/Loader/Loader.stories.mdx

```diff
@@ -10,14 +11,27 @@ import { Loader } from './Loader';
 
 # Loader
 
-This is the doc of the `Loader` component
+Loaders indicate some data is currently loading where content usually takes place. They help to notify users that loading is ongoing.
 
-## Loader
+**Best** **practices**
 
-Description...
+- Once the content appears, the loader disappears.
+- If the trigger only concerns a portion of a page, the loader should be placed in that part.
+- Loaders should be scaled down if they aimed to be placed in a small portion of a page.
```

Comment:
We don't have a spec for it nor a side to adjust. We can't "scale it on demand" :p

Reply:
Alright, I thought something "easy" to do, let's remove it then

Reply:
And I won't bring the consistency argument with you 😝